### PR TITLE
dec 2021 comm call on automation

### PR DESCRIPTION
**Pull request by stefaniebutland**

### CODE REVIEW

File: content/commcalls/2021-12-07/index.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+# vimeo:
+# - id: '518761488'
+resources:
+  - url: /software-review/
+    name: rOpenSci Software Peer Review
+    details: 
+  - url: https://joss.theoj.org/
+    name: Journal of Open Source Software
+    details: 
+  - url: https://github.com/ropensci-org/buffy
```
Comment (maelle) [inline content/commcalls/2021-12-07/index.md:31]:
```suggestion
  - url: https://buffy-ropensci.readthedocs.io/en/latest/
```

File: content/commcalls/2021-12-07/index.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    name: Buffy
+    details: is a service to provide a bot helping scientific journals manage submission reviews. Automates common editorial tasks like those needed by JOSS or rOpenSci.
```
Comment (maelle) [inline content/commcalls/2021-12-07/index.md:33]:
```suggestion
    details: is a service to provide a bot helping scientific journals manage submission reviews. Automates common editorial tasks like those needed by JOSS or rOpenSci.
  - url: https://buffy-ropensci.readthedocs.io/en/latest/
    name: rOpenSci version of Buffy
    details: holding our specific configuration and commands.
```

File: content/commcalls/2021-12-07/index.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+deets: |
+    Meeting ID: 939 9725 9411
+    
+    Passcode: 615476
+zoomurl: https://zoom.us/j/93997259411?pwd=Qmp5RHZlVHc0NGI1NVdnZTMvZkVkdz09
+---
+
+Software peer-review involves coordinating and tracking many moving parts: software submissions, testing and diagnostics, assignment of editors and reviewers, and logging the progression of submissions through revisions and acceptance.
+
+On this call we will discuss how rOpenSci has worked with The [Journal of Open Source Software](https://joss.theoj.org/) to extend JOSS’s approach of [chatops-driven publishing](https://www.arfon.org/chatops-driven-publishing) into a new GitHub chat-bot that manages our editorial process: assigning tasks, tagging issues, running tests on software submissions, and returning reports to reviewers and editors, all from the comfort of a GitHub issue comment. Chat-ops automation has been critical to JOSS’s success, and shows huge promise for automating even more tedious tasks like frequent software checks and compliance with best practices.
```
Comment (maelle) [inline content/commcalls/2021-12-07/index.md:68]:
```suggestion
On this call we will discuss how rOpenSci has worked with The [Journal of Open Source Software](https://joss.theoj.org/) to extend JOSS’s approach of [chatops-driven publishing](https://www.arfon.org/chatops-driven-publishing) into a new GitHub chat-bot that manages our editorial process: assigning tasks, tagging issues, running tests on software submissions, and returning reports to reviewers and editors, logging reviews in an external (Airtable) database, all from the comfort of a GitHub issue comment. Chat-ops automation has been critical to JOSS’s success, and shows huge promise for automating even more tedious tasks like frequent software checks and compliance with best practices.
```

**Review by maelle — APPROVED** [review]
Thank you @stefaniebutland!

**Review by arfon — APPROVED** [review]
LGTM!

**Pull request merged**